informer/informer_integration_basic_test.go

```diff
@@ -93,8 +95,8 @@ func Test_Informer_Integration_Basic(t *testing.T) {
 			testAssertCROWithID(t, e, objectIDOne, objectIDTwo)
 		}
 
-		d := time.Now().Sub(start)
-		if d.Seconds() > 10.1 {
+		d := time.Since(start)
+		if !durationEquals(10*time.Second, d, timeDelta) {
```

Comment:
Can we be sure this works all the time? IIRC I used some 100 milliseconds threshold  because of some weird timing issues.

Reply:
It worked on my local machine. But this is a hefty one. I guess on slow CI container it may take way longer. Even a few times longer from my experience.

> I would also like to add the integration test magic to circle. Do you know how to do it? I do not right now.

I don't know it either. We may use some parts of e2e testing probably. @fgimenez should know more.